frontend/public/components/edit-yaml.jsx

```diff
@@ -545,18 +550,25 @@ export const EditYAML_ = connect(stateToProps)(
                             type="submit"
                             variant="secondary"
                             id="reload-object"
+                            data-test="reload-object"
                             onClick={() => this.reload()}
                           >
                             Reload
                           </Button>
                         )}
-                        <Button variant="secondary" id="cancel" onClick={() => this.onCancel()}>
+                        <Button
+                          variant="secondary"
+                          id="cancel"
+                          data-test="cancel"
+                          onClick={() => this.onCancel()}
+                        >
                           Cancel
                         </Button>
                         {download && (
                           <Button
                             type="submit"
                             variant="secondary"
+                            data-test="download"
```

Comment:
Are we actually testing this? We should only add `data-test` attributes where we need them.

Reply:
fixed, removed